store/sqlstore/team_store.go

```diff
@@ -678,14 +678,19 @@ func (s SqlTeamStore) GetMember(teamId string, userId string) (*model.TeamMember
 	return dbMember.ToModel(), nil
 }
 
-func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restrictions *model.ViewUsersRestrictions) ([]*model.TeamMember, *model.AppError) {
+func (s SqlTeamStore) GetMembers(teamId string, offset int, limit int, restrictions *model.ViewUsersRestrictions, teamMembersGetOptions *model.TeamMembersGetOptions) ([]*model.TeamMember, *model.AppError) {
 	query := s.getTeamMembersWithSchemeSelectQuery().
 		Where(sq.Eq{"TeamMembers.TeamId": teamId}).
 		Where(sq.Eq{"TeamMembers.DeleteAt": 0}).
-		OrderBy("UserId").
 		Limit(uint64(limit)).
 		Offset(uint64(offset))
 
+	if teamMembersGetOptions.Sort == "Username" {
+		query = query.LeftJoin("Users ON TeamMembers.UserId = Users.Id").Where(sq.Eq{"Users.DeleteAt": 0}).OrderBy("Username")
```

Comment:
Seems unexpected to bury `Users.DeleteAt` as part of the `Sort`. Should we have a separate `ExcludeDeleted` flag?

Reply:
Thanks for pointing this out. I've added and added unit tests